OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -168,12 +168,27 @@ public override bool YieldKeyboardFocus()
 
 		public override bool HandleKeyPress(KeyInput e)
 		{
-			switch (e.Key)
```

Comment:
When i tried switch I got "Error	1	A switch expression or case label must be a bool, char, string, integral, enum, or corresponding nullable type	"